closed/OpenJ9.gmk

```diff
@@ -187,6 +187,27 @@ stage-j9 : \
 	@$(ECHO) Staging OpenJ9 OMR in $(OUTPUT_ROOT)/vm
 	$(call openj9_copy_tree,$(OUTPUT_ROOT)/vm/omr,$(OPENJ9OMR_TOPDIR))
 
+OPENJ9_VERSION_VARS := \
+	COMPILER_VERSION_STRING \
+	HOTSPOT_TARGET_OS \
+	OPENJDK_TARGET_CPU_BITS \
+	OPENJDK_TARGET_CPU_OSARCH \
+	PRODUCT_NAME \
+	USERNAME \
+	VERSION_STRING \
+	#
+
+OPENJ9_VERSION_SCRIPT := \
+	$(foreach var,$(OPENJ9_VERSION_VARS),-e 's:@${var}@:$(value $(var)):g')
+
+$(OUTPUT_ROOT)/vm/util/openj9_version_info.h : $(SRC_ROOT)/closed/openj9_version_info.h.in
+	@$(MKDIR) -p $(@D)
```

Comment:
The directory should already be present from a successful stage-j9 target execution so the mkdir shouldn't be required here.

Reply:
For everyone's benefit: you can assume the actions of another target have occurred _only_ if that target is a (possibly indirect) prerequisite. Make could legally build prerequisites in any order it sees fit (and will likely do so when allowed to do things in parallel) it is only bound by dependencies.